gateway-provider-security-jwt/src/main/java/org/apache/knox/gateway/provider/federation/jwt/filter/JWTFederationFilter.java

```diff
@@ -80,6 +92,10 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
       // what follows the bearer designator should be the JWT token being used to request or as an access token
       wireToken = header.substring(BEARER.length());
     }
+    else if((((HttpServletRequest) request).getHeader(HEADER_TOKEN_NAME))!= null) {
```

Comment:
Sorry - This is going to be super nitpicky - can this be a separate Jira/PR for this change for Hive? I want to call it out separately since its not really the main part of this change. I also wonder why a header with a space can't be passed. That seems like a weird Hive JDBC restriction that I haven't seen before.

Reply:
okay , Let me remove as part of this Pull request . for Hive I  will work  different PR